### ISSUE TITLE
Add Elmah.Beta.SQLCompact to NuGet

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#209]
```
It'd be nice if we had either

Elmah.Beta
Elmah.Beta.SQLCompact (just the web.config changes)

OR, even better, if Elmah 1.2 in NuGet made SQL Compact the default. I'd be 
happy to help with packaging/releasing whatever. 

Basically just add these to the Elmah 1.2 NuGet packaged web.config, add a 
dependancy to SQLServerCompact in the NuSpec, and put it in NuGet.

  <connectionStrings>
    <add name="ElmahSQLCE" connectionString="data source=|DataDirectory|\Elmah.sdf" />
  </connectionStrings>
  <elmah>
    <errorLog type="Elmah.SqlCompactErrorLog, Elmah" connectionStringName="ElmahSQLCE" />
  </elmah>

It would make the out of the box experience nicer (not memory based) and 
survive recycles.

Let me know if I can help. We could make a bunch of web.config defaults like 
elmah.SQLlite, etc.
```

Original issue reported on code.google.com by `Glucopi...@gmail.com` on 22 Feb 2011 at 6:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `jamesdriscoll71` on 22 Feb 2011 at 10:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jamesdriscoll71` on 1 Mar 2011 at 10:21
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
> Let me know if I can help.

The initial set of packages are checked into branch RB-1.2-NuGet[1]. Would be 
great if you can review, test and provide feedback. If you'd like to suggest 
modifications directly, feel free to submit patches.

[1] http://code.google.com/p/elmah/source/browse/#svn%2Fbranches%2FRB-1.2-NuGet

```

Original comment by `azizatif` on 1 Mar 2011 at 11:45
- Added labels: **Component-Scripts**, **Milestone-Release1.2**

**Comment by GoogleCodeExporter** [comment]
```
I checked out the branch and ran pack.cmd in the root. The batch script seemed 
to have run fine except for one glitch:

> Could not find a part of the path
> 'C:\ELMAH-1.2.nuget\bin\net-2.0\release\'.

See the attached packlog.txt for full details. I think this is because pack.cmd 
expects that the solution has already been built. If that's true, it should run 
src\Solutions\build.cmd prior to packaging.
```

Original comment by `azizatif` on 2 Mar 2011 at 12:23

Attachments:
- [packlog.txt](https://storage.googleapis.com/google-code-attachments/elmah/issue-209/comment-4/packlog.txt)

**Comment by GoogleCodeExporter** [comment]
```
Resolved - pack.cmd now forces a build before starting to package
```

Original comment by `jamesdriscoll71` on 2 Mar 2011 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
Fixed in ELMAH 1.2 NuGet branch
New task has been created #210 to improve this functionality with MsBuild
```

Original comment by `jamesdriscoll71` on 10 Mar 2011 at 2:00
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Re-opening this issue as “Started” since a merge back into 1.2 release 
branch is still pending.
```

Original comment by `azizatif` on 10 Mar 2011 at 10:23
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Moved to its own tree in r830 rather than merging to 1.2 release branch.
```

Original comment by `azizatif` on 16 May 2011 at 9:02
- Changed state: **Fixed**
